content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -28,6 +26,16 @@ containers, the timezone set for the kube-controller-manager container determine
 that the cron job controller uses.
 {{< /caution >}}
 
+{{< caution >}}
+The [v1 CronJob API](/docs/reference/kubernetes-api/workload-resources/cron-job-v1/)
+does not officially support setting timezone as explained above.
```

Comment:
"as explained above" ?

Reply:
I think this is OK to merge with the existing wording but a follow-up to clarify would be welcome.

Reply:
above is a caution about cronjob using timezone from kube-controller-manager's host, that's what I've meant with it